### PR TITLE
chore: Add example of using bitcoin_get_block_headers to basic_bitcoin

### PR DESCRIPTION
**Pull request by dragoljub-duric**
A new endpoint is added to Bitcoin canister, this PR shows its desired usage.

### CODE REVIEW

File: rust/basic_bitcoin/README.md [hunk 1]
```diff
@@ -214,6 +214,26 @@ it sent to the network. You can track the status of this transaction using a
 transaction has at least one confirmation, you should be able to see it
 reflected in your current balance.
 
+## Step 6: Retrieving block headers
+
+You can also get range of Bitcoin block headers by using the `get_block_headers`
```
Comment (dsarlis) [inline rust/basic_bitcoin/README.md:219]:
```suggestion
You can also get a range of Bitcoin block headers by using the `get_block_headers` 
```

File: rust/basic_bitcoin/README.md [hunk 2]
```diff
@@ -214,6 +214,26 @@ it sent to the network. You can track the status of this transaction using a
...
+
+```bash
+dfx canister --network=ic call basic_bitcoin get_block_headers "(10: nat32)"
+```
+or replace `0` and `11` with your desired start and end height respectively:
+```bash
+dfx canister --network=ic call basic_bitcoin get_block_headers "(0: nat32, 11: nat32)"
+```
+
+Checking the balance of a Bitcoin address relies on the [bitcoin_get_block_headers](https://internetcomputer.org/docs/current/references/ic-interface-spec/#ic-bitcoin_block_headers) API.
```
Comment (dsarlis) [inline rust/basic_bitcoin/README.md:235]: Is this really true? Seems like a copy-paste error to me.

File: rust/basic_bitcoin/src/basic_bitcoin/src/lib.rs [hunk 3]
```diff
@@ -52,6 +53,31 @@ pub async fn get_utxos(address: String) -> GetUtxosResponse {
     bitcoin_api::get_utxos(network, address).await
 }
 
+pub type Height = u32;
+pub type BlockHeader = Vec<u8>;
+
+/// A request for getting the block headers from a given height.
```
Comment (dsarlis) [inline rust/basic_bitcoin/src/basic_bitcoin/src/lib.rs:59]:
```suggestion
/// A request for getting the block headers for a given height range.
```

**Review by dsarlis — APPROVED** [review]

File: rust/basic_bitcoin/README.md (hunk 2 above)
Comment (dragoljub-duric) [reply rust/basic_bitcoin/README.md:235]: It is not, I will remove it.

**Pull request merged**